index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    />
+    <title>Thump a Trump</title>
+    <link
+      rel="stylesheet"
+      href="normalize.css"
+      type="text/css"
+    />
+    <link
+      rel="stylesheet"
+      href="style.css"
+      type="text/css"
+    />
+    <link
+      rel="stylesheet"
+      href="animations.css"
+      type="text/css"
+    />
+  </head>
+  <body>
+    <header>
+      <h1>Thump a Trump</h1>
+    </header>
+
+    <section class="menuScreen">
+      <img class="trumpy" src="images/trump01.svg" alt="" />
+
+      <a href="game.html">
+        <div class="startBtn">
+          <button>
+            <img
+              src="images/playSolid.svg"
+              class="startImg"
+            />START
+          </button>
+        </div>
+      </a>
+
+      <div class="instructions">
+        <h3>How to play</h3>
+        <ol>
+          <li>Press start</li>
+          <li>Thump Trump</li>
+          <li>Trumps appear for one second</li>
+          <li>
+            You have 30 seconds to MAKE AMERICA GREAT AGAIN!
+          </li>
```

Comment:
Inconsistent formatting on these lines, the bottom list item is broken down onto multiple lines. Could you just pick either or?